Story/OnlineUIComponents/SpectatorOverlay.cs

```diff
@@ -49,8 +49,10 @@ public PlayerButton(SpectatorOverlay menu, OnlinePhysicalObject opo, Vector2 pos
                 this.button.OnClick += (_) =>
                 {
                     this.button.toggled = !this.button.toggled;
-                    overlay.spectatee = this.button.toggled ? opo.apo as AbstractCreature : null;
-                    OnlineManager.mePlayer.isActuallySpectating = overlay.spectatee == null || !this.player.isMine;
+                    overlay.spectatee = this.button.toggled ? this.player.apo as AbstractCreature : null;
+                    bool result = !this.player.isMine ? true : overlay.spectatee == null ? false : false;
```

Comment:
Maybe we could just simplify to 

OnlineManager.mePlayer.isActuallySpectating = !this.player.isMine? What's the thinking behind == null?